schemas/dependencies/jenkins-instance-1.yml

```diff
@@ -50,6 +50,39 @@ properties:
       required:
       - name
       - keep_hours
+  workerFleets:
+    type: array
+    items:
+      type: object
+      additionalProperties: false
+      properties:
+        account:
+          "$ref": "/aws/tenant_accounts-1.yml#/properties/account"
```

Comment:
Do we actually need this, or since we have to match the ASG it is redundant?

Reply:
well, we can have two ASGs with the same name in different accounts, so this is pertinent.